src/model/schema.js

```diff
@@ -1165,13 +1163,25 @@ export class SchemaContext {
 
 	/**
 	 * Returns new SchemaContext instance with additional items created from provided definition.
+	 * Definition can be:
+	 *
+	 * 		const ctx = new SchemaContext( element ); // [ 'elementParent', 'element' ]
+	 *
+	 * 		// A node.
+	 * 		const ctx2 = ctx.concat( element.getChild( 0 ) ); // [ 'elementParent', 'element', 'elementChild' ]
```

Comment:
I would write:
```js
const fooElement = writer.createElement( 'foo' );
const context2 = context.concat( element ); // [ 'elementParent', 'element', 'foo' ]
```
First, becase using element which is already in the tree is strange case for the concat method, second, to make it clear where the name come from.

Reply:
This is to show that element that already has a parent will be added without this parent.

Reply:
I would write it as a note "If the element has a parent, only the element will be added to the new context (the partent will not be included)." The sample does not show it well, especially when you have some parents in the outputs.

Reply:
If I would see such sample I would ask myself: why should I do such thing? Isn't it better to do `new SchemaContext( element.getChild( 0 ) );`?

Reply:
Changed.